src/hotspot/share/opto/output.cpp

```diff
@@ -22,6 +22,12 @@
  *
  */
 
+/*
+ * This file has been modified by Loongson Technology in 2022. These
+ * modifications are Copyright (c) 2022, Loongson Technology, and are made
```

Comment:
I'm afraid this part of change is unnecessary.
Maybe, you can just remove it, right?

Reply:
Thanks for the reminder.